src/mame/video/pgm.cpp

```diff
@@ -669,5 +669,6 @@ WRITE_LINE_MEMBER(pgm_state::screen_vblank_pgm)
 	{
 		/* first 0xa00 of main ram = sprites, seems to be buffered, DMA? */
 		memcpy(m_spritebufferram.get(), m_mainram, 0xa00);
+		m_maincpu->set_input_line(6, HOLD_LINE);
```

Comment:
Why was this added?

Reply:
Some driver has converted VBlank interrupt in screen_vblank callbacks; so I converted VBlank interrupt in here to.

Reply:
Oh sorry, I missed where it was removed from mconfig.